src/components/BeforeGame.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import Input from './Input';
+import Button from './Button';
+
+import boom from '../assets/img/boom.png';
+
+
+class BeforeGame extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onInput1Change = this.onInput1Change.bind(this);
+        this.onInput2Change = this.onInput2Change.bind(this);
+        this.onBeginGame = this.onBeginGame.bind(this);
+
+        this.state = {
+            player1: '',
+            player2: ''
+        }
+    }
+
+    onInput1Change(e) {
```

Comment:
I would name it as onPlayerOneChange (avoid using numbers in names, and it is good to say what exactly the function do, so here we are changing player 1 name).
The same for second function.